tiflash/tiflash-supported-pushdown-calculations.md

```diff
@@ -150,3 +155,52 @@ SHOW WARNINGS;
+ALTER TABLE t SET TIFLASH REPLICA 1;
+INSERT INTO t VALUES(1,"foo"),(2,"bar"),(3,"bar foo"),(10,"foo"),(20,"bar"),(30,"bar foo");
+
+EXPLAIN SELECT id, ROW_NUMBER() OVER (PARTITION BY id > 10) FROM t;
++----------------------------------+----------+--------------+---------------+---------------------------------------------------------------------------------------------------------------+
+| id                               | estRows  | task         | access object | operator info                                                                                                 |
++----------------------------------+----------+--------------+---------------+---------------------------------------------------------------------------------------------------------------+
+| TableReader_30                   | 10000.00 | root         |               | MppVersion: 1, data:ExchangeSender_29                                                                         |
+| └─ExchangeSender_29              | 10000.00 | mpp[tiflash] |               | ExchangeType: PassThrough                                                                                     |
+|   └─Projection_7                 | 10000.00 | mpp[tiflash] |               | test.t.id, Column#5, stream_count: 4                                                                          |
+|     └─Window_28                  | 10000.00 | mpp[tiflash] |               | row_number()->Column#5 over(partition by Column#4 rows between current row and current row), stream_count: 4  |
+|       └─Sort_14                  | 10000.00 | mpp[tiflash] |               | Column#4, stream_count: 4                                                                                     |
+|         └─ExchangeReceiver_13    | 10000.00 | mpp[tiflash] |               | stream_count: 4                                                                                               |
+|           └─ExchangeSender_12    | 10000.00 | mpp[tiflash] |               | ExchangeType: HashPartition, Compression: FAST, Hash Cols: [name: Column#4, collate: binary], stream_count: 4 |
+|             └─Projection_10      | 10000.00 | mpp[tiflash] |               | test.t.id, gt(test.t.id, 10)->Column#4                                                                        |
+|               └─TableFullScan_11 | 10000.00 | mpp[tiflash] | table:t       | keep order:false, stats:pseudo                                                                                |
++----------------------------------+----------+--------------+---------------+---------------------------------------------------------------------------------------------------------------+
+9 rows in set (0.0073 sec)
+
+```
+
+In this output, you can see that the `Window` operation has a value of `mpp[tiflash]` in the `task` column indicating it was able to push down the `ROW_NUMBER() OVER (PARTITION BY id > 10)` operation to TiFlash.
```

Comment:
In this phrase `indicating it was able to push down the `ROW_NUMBER() OVER (PARTITION BY id > 10)` operation to TiFlash`, what does `it` mean?

Reply:
Yes, that's a good suggestion.